tests/net/rpl/src/main.c

```diff
@@ -299,22 +280,19 @@ static bool test_rpl_mcast_addr(void)
 
 	ret = net_rpl_is_ipv6_addr_mcast(&rpl_mcast);
 	if (!ret) {
-		TC_ERROR("RPL multicast address check failed.\n");
-		return false;
+		zassert_true(0, "RPL multicast address check failed.");
 	}
 
 	net_rpl_create_mcast_address(&addr);
 
 	ret = net_rpl_is_ipv6_addr_mcast(&addr);
 	if (!ret) {
-		TC_ERROR("Generated RPL multicast address check failed.\n");
-		return false;
+		zassert_true(0, "Generated RPL multicast "
```

Comment:
don't split strings even if they go past 80 characters, it is much harder to grep for them.
checkpatch makes an exception for strings like this.

Reply:
fixed